### PR TITLE
1023 Recording Callback event example request format

### PR DESCRIPTION
**Pull request by nirm2009**

### CODE REVIEW

File: cloud-recording/reference/rest-api/rest-api-overview.md [hunk 1]
```diff
@@ -522,7 +525,7 @@ eventType 43 indicates that the state of the audio stream has changed, and `deta
 - `msgName: `String. The message name, `"web_recorder_reload"`:
 - `reason: `String. The reason for the page reload:
   - `"audio_silence"`: Audio loss issue.
-  -  `"page_load_timeout"`: Web page load timeout. The callback returns this field only under the following conditions: the web page load detection function is enabled, and the web page load timeout. See [Web page load detection](../../develop/webpage-load-timeout).
+  - `"page_load_timeout"`: Web page load timeout. The callback returns this field only under the following conditions: the web page load detection function is enabled, and the web page load timeout. See [Web page load detection](../../develop/webpage-load-timeout).
```
Comment (atovpeko) [inline cloud-recording/reference/rest-api/rest-api-overview.md:528]: 1. Please capitalize the first letter after the colon.
2. Could you rephrase 'The callback returns this field only under the following conditions: the web page load detection function is enabled, and the web page load timeout.' so that both parts after the colon are more aligned? I'd say the second one needs a verb (for example, and there is a web page load timeout)

**Review by atovpeko — CHANGES_REQUESTED** [review]

File: cloud-recording/reference/rest-api/rest-api-overview.md [hunk 2]
```diff
@@ -525,7 +525,7 @@ eventType 43 indicates that the state of the audio stream has changed, and `deta
 - `msgName: `String. The message name, `"web_recorder_reload"`:
 - `reason: `String. The reason for the page reload:
   - `"audio_silence"`: Audio loss issue.
-  - `"page_load_timeout"`: Web page load timeout. The callback returns this field only under the following conditions: the web page load detection function is enabled, and the web page load timeout. See [Web page load detection](../../develop/webpage-load-timeout).
+  - `"page_load_timeout"`: Web page load timeout; the callback returns this field only if the web page load detection function is enabled and there is a web page load timeout. See [Web page load detection](../../develop/webpage-load-timeout).
```
Comment (atovpeko) [inline cloud-recording/reference/rest-api/rest-api-overview.md:528]: It was good with a full stop. We try to avoid ; in documentation.

**Review by atovpeko — CHANGES_REQUESTED** [review]

**Comment by billy-the-fish** [comment]
Are all the review updates done and we are ready to merge?

**Review by atovpeko — APPROVED** [review]

**Comment by atovpeko** [comment]
Good to go
